src/Illuminate/Support/Str.php

```diff
@@ -1179,8 +1178,8 @@ public static function repeat(string $string, int $times)
      */
     public static function replaceArray($search, $replace, $subject)
     {
-        if ($replace instanceof Traversable) {
-            $replace = (new Collection($replace))->all();
+        if (Arr::arrayable($replace)) {
```

Comment:
Not sure if this is an improvement as this now calls a function and then has to check cases that are not needed here

Reply:
`Arr::arrayable` check could be omitted because `Arr::from` passes through arrays anyway.